libs/Transisthor/TransisthorECSLogic/Server/Systems/SendToClient.hpp

```diff
@@ -43,6 +42,7 @@ struct SendToClient : public ecs::System {
         (void)networkId;
         (void)entity;
         (void)clientIdList;
+        (void)world;
```

Comment:
Because add this point (when the types list is empty), it nothing must be done. It's used in a recursive case. The vois is in order to have none unused variables.

Reply:
Because add this point (when the types list is empty), it nothing must be done. It's used in a recursive case. The vois is in order to have none unused variables.